cloudstate-kotlin-support/src/main/kotlin/io/cloudstate/kotlinsupport/ReflectionHelper.kt

```diff
@@ -0,0 +1,22 @@
+package io.cloudstate.kotlinsupport
+
+class ReflectionHelper {
```

Comment:
`final`?

Reply:
In Kotlin, all the classes are final by default
This is a reason why other classes are not final. Strength of habit